app/src/main/java/org/mozilla/focus/utils/Features.kt

```diff
@@ -9,6 +9,9 @@ package org.mozilla.focus.utils
  */
 object Features {
     const val SEARCH_TERMS_OR_URL: Boolean = true
+    const val SHOULD_SHOW_CFR_FOR_SHIELD_TOOLBAR_ICON: Boolean = false
+    const val SHOULD_SHOW_TOOLTIP_FOR_PRIVACY_SECURITY_SETTINGS_SCREEN: Boolean = false
```

Comment:
I wonder if we should consider enabling these for debug versions so they can also be tested. @ionutbedregeanu what do you think?

Reply:
Agree, QA team will thank us

Reply:
Done